examples/demo13.py

```diff
@@ -54,308 +55,302 @@
+            # The rng for photon shooting should be different for each filter.
+            phot_rng = galsim.UniformDeviate(seed + 1 + i_obj + ifilter*nobj)
+
+            # We'll deal with this below.  The config processing calculates this before the
+            # position, so to get the same answers, we need to do the same here.
+            p = obj_rng()
+
+            # Pick a random position in the image to draw it.
+            # If we had a real galaxy catalog with positions in terms of RA, Dec we could use
+            # wcs.toImage() to find where those objects should be in terms of (x, y).
+            # Note that we could use wcs.toWorld() to get the (RA, Dec) for these (x, y) positions
+            # if we wanted that information, but we don't need it.
+            x = obj_rng() * roman.n_pix
+            y = obj_rng() * roman.n_pix
+            image_pos = galsim.PositionD(x,y)
+            logger.debug('Position = %s',image_pos)
+
+            # Now decide which of our three kinds of objects we want to draw:
+            # 80% faint galaxy
+            # 10% star
+            # 10% bright galaxy
+            if p < 0.8:
+                # Faint galaxy
+                logger.debug('Faint galaxy')
+
+                # Select a random galaxy from the catalog.
+                obj = cat1.makeGalaxy(chromatic=True, gal_type='parametric', rng=obj_rng)
+                logger.debug('galaxy index = %s',obj.index)
+
+                # Rotate the galaxy randomly
+                theta = obj_rng() * 2 * np.pi * galsim.radians
+                logger.debug('theta = %s',theta)
+                obj = obj.rotate(theta)
+
+            elif p < 0.9:
+                # Star
+                logger.debug('Star')
+
+                # Use a log-normal distribution for the stellar fluxes.  This gives a few very
+                # bright stars, but not too many.
+                # cf. https://en.wikipedia.org/wiki/Log-normal_distribution
+                # mu_x, sigma_x are the target mean, std.dev.
+                # mu,sigma are the parameters of the function.
+                mu_x = 1.e5
+                sigma_x = 2.e5
+                mu = np.log(mu_x**2 / (mu_x**2+sigma_x**2)**0.5)
+                sigma = (np.log(1 + sigma_x**2/mu_x**2))**0.5
+                gd = galsim.GaussianDeviate(obj_rng, mean=mu, sigma=sigma)
+                flux = np.exp(gd())
+                logger.debug('flux = %s',flux)
+
+                # Normalize the SED to have this flux in the Y band.
+                sed = vega_sed.withFlux(flux, y_bandpass)
+
+                obj = galsim.DeltaFunction() * sed
+
+            else:
+                # Bright galaxy
+                logger.debug('Bright galaxy')
+
+                obj = cat2.makeGalaxy(chromatic=True, gal_type='parametric', rng=obj_rng)
+                logger.debug('galaxy index = %s',obj.index)
+
+                # Scale up the area by a factor of 2, and the flux by a factor of 4.
```

Comment:
Why?

Reply:
Just for visual impact to get a few more big, bright galaxies that can try to represent brighter galaxies than the bright limit of the 25.2 catalog.

Reply:
OK, might be worth mentioning.

Reply:
I think I did.  The update has:
```
# This is not necessarily physical, but it is intended to add some more big,                
# bright galaxies to the scene to make the final image a bit more interesting.
```
Does that seem ok?

Reply:
Yes, thanks.